### PR TITLE
Add a control for sharpness in CAS upscaling

### PR DESCRIPTION
**Pull request by apazylbe**

### CODE REVIEW

File: src/app/AppCore.h [hunk 1]
```diff
@@ -167,6 +167,10 @@ struct GpuTimerInfo {
   uint64_t end_time;
 };
 
+struct CASUpscalingInfo {
```
Comment (ndeakin) [inline src/app/AppCore.h:170]: nit: could "parameters" or "params" or something to that a affect be more clear than "info" perhaps?

**Comment by ndeakin** [comment]
If I understand correctly, this is intended to add the control for sharpness, but it doesn't cause it to have any affect yet - correct?

**Comment by robbiesri** [comment]
> If I understand correctly, this is intended to add the control for sharpness, but it doesn't cause it to have any affect yet - correct?

Actually, it does. Take a look at how `UpdateCASConstants` is implemented: https://github.com/googlestadia/PorQue4K/blob/209ba90a35f05df0be68df586c9e184183ed8263/src/app/CAS.cpp#L27

It packs the `sharpness` arg into the constant buffer that the CAS shader uses.

File: src/app/AppCore.h (hunk 1 above)
Comment (robbiesri) [reply src/app/AppCore.h:170]: I agree with Nick. CASUpscalingParams or CASDynamicParams would be cool.

**Comment by ndeakin** [comment]
ahhhh got it, makes sense, thanks for the clarification!

File: src/app/AppCore.h (hunk 1 above)
Comment (apazylbe) [reply src/app/AppCore.h]: Done, thanks!

**Pull request merged**